lib/models/builder.js

```diff
@@ -36,8 +36,21 @@ module.exports = Task.extend({
     this.trapSignals();
   },
 
+  /**
+    Determine whether the output path is safe to delete. If the outputPath
+    appears anywhere in the parents of the project root, the build would 
+    delete the project directory. In this case return `false`, otherwise
+    return `true`.
+  */
   canDeleteOutputPath: function(outputPath) {
-    return this.project.root.indexOf(outputPath) === -1;
+    var rootPathParents = [this.project.root];
+    var parsedPath = path.parse(this.project.root);
```

Comment:
I can take a look...I'm really just using dir from the parsed path to make sure I get properly uniform and normalized paths (unfortunately path.normalize() wasn't cutting it, and that parents module that looked cool was also normalizing poorly, in a different way from path.normalize()) so path.dirname() might substitute just fine. If I have trouble with that, I'll look at backporting parse.